boards/riscv/mpfs_icicle/support/mpfs250t.resc

```diff
@@ -0,0 +1,17 @@
+:name: MPFS-ICICLE-KIT
+:description: This script is prepared to run Zephyr on a PolarFire SoC Icicle Kit RISC-V board.
+
+$name?="MPFS-ICICLE-KIT"
+
+using sysbus
+mach create $name
+machine LoadPlatformDescription @platforms/boards/mpfs-icicle-kit.repl
+
+showAnalyzer uart
+cpu PerformanceInMips 80
```

Comment:
This should be `e51` I think, as `cpu` is not defined in [polarfire-soc.repl](https://github.com/renode/renode/blob/master/platforms/cpus/polarfire-soc.repl)